pom.xml

```diff
@@ -101,6 +101,12 @@ all modules and does not describe the project structure per-subproject.
     <version>2.0.0-SNAPSHOT.242</version>
     <scope>test</scope>
   </dependency>
+  <dependency>
+    <groupId>io.spine.tools</groupId>
+    <artifactId>spine-logging-testlib</artifactId>
```

Comment:
How comes this dependency suddenly appears here now? :)

Reply:
I've taken `master`, built it, and `POM.xml` was updated with this dependency. Maybe something related to local caches or stale POM files of the published artifacts.

Reply:
This is a direct dependency to use `tapConsole { }` I added in #859 as a review suggestion (so, after the bumped version and updated reports).

Hence, that PR was merged to `master` with a stale POM. Sooner or later, we should move this routine to CI :)

>> I followed the clues… right back to me.